source/lib/projectz.js

```diff
@@ -386,8 +386,9 @@ class Projectz {
 
 		// Extract repository information
 		/* eslint no-magic-numbers: 0 */
-		let repo = this.mergedPackageData.repository.url || null
-		const githubMatch = (this.mergedPackageData.repository.url || this.mergedPackageData.homepage).match(/github\.com\/(.+?)(?:\.git|\/)?$/)
+		let repo = this.mergedPackageData.repository || null
+		const githubMatch = (this.mergedPackageData.repository.url || this.mergedPackageData.homepage).match(/github\.com\/(.+?)(?:\.git|\/)?$/) ||
+			(this.mergedPackageData.repository.match(/^(.+\/.+)$/) && !this.mergedPackageData.repository.match(/^(.+:.+\/.+)$/))
```

Comment:
Couldn't do this the other way. If you have a better idea how to match `user/repo` but not `(bitbucket|gist|gitlab):user/repo`, please tell me.